docs/data-sources/vpc_subnet.md

```diff
@@ -61,6 +61,8 @@ In addition to all arguments above, the following attributes are exported:
 
 * `ipv6_enable` - Whether the IPv6 is enabled.
 
+* `ipv6_subnet_id` - The ID of the IPv6 subnet. If the subnet is an IPv4 subnet, this parameter is not returned.
```

Comment:
can we remove `If the subnet is an IPv4 subnet, this parameter is not returned.` ? I think it is unnecessary?

Reply:
yeah, other IPv6 parameters also have this description.
None of them make sense.